lms/static/scripts/frontend_apps/common/dom.js

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
I didn't review the additions to `frontend_apps/common` because the canonical location for that is the client repository.